.github/workflows/tests-on-push.yml

```diff
@@ -91,6 +91,13 @@ jobs:
         run: poetry exec e2e-tests
       - name: stop stack  # see if this command can run (it isn't tested anywhere else)
         run: poetry run dsp-tools stop-stack
+      - name: make logging file available
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: logging.log
+          path: /home/runner/.dsp-tools/logging.log
+          compression-level: 0
```

Comment:
The result looks like this, and will also be available if the e2e tests fail: https://github.com/dasch-swiss/dsp-tools/actions/runs/8379830791/job/22947665025#step:7:27